lib/command.js

```diff
@@ -445,6 +446,22 @@ const commands = {
 						});
 				});
 		});
+	},
+
+	checkout(event, statusBar){
+		return getFiles(event.target).then(([files, root]) => {
+			return git.branches(root)
+			.then((branches) => {
+				return new checkoutDialog().activate(branches)
+			})
+			.then(([branch]) => {
+				return git.checkout(root, branch)
+				.then((result) => {
+					refreshAtom(root);
+					return result;
```

Comment:
I'm outputting the result of the git command.  On other commands you've just output a simple success message.

My personal preference is to see the output from git itself, but this isn't consistent with the other command results.